features/app/constants.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Available application screens/routes.
+ * 
+ * @enum {string}
+ */
+export const APP_SCREEN = {
+    HOME: 'home',
```

Comment:
Our style guide requires an entity to be named with 1 name in its multiple occurrences. If HOME refers to features/welcome and its WelcomePage, the name here should be WELCOME as well.